lib/runners/browser_test_runner.js

```diff
@@ -128,6 +128,14 @@ module.exports = class BrowserTestRunner {
     socket.on('all-test-results', this.onAllTestResults.bind(this));
     socket.on('after-tests-complete', this.onAfterTests.bind(this));
 
+    const additionalBrowserSocketEvents = this.launcher.config.get('additionalBrowserSocketEvents');
```

Comment:
We should add documentation to https://github.com/choheekim/testem/blob/master/docs/config_file.md
to explain the usage of this option.